x-pack/plugins/apm/server/lib/anomaly_detection/create_anomaly_detection_jobs.ts

```diff
@@ -92,8 +92,9 @@ async function createAnomalyDetectionJob({
       query: {
         bool: {
           filter: [
-            { term: { [PROCESSOR_EVENT]: ProcessorEvent.transaction } },
-            { exists: { field: TRANSACTION_DURATION } },
+            { term: { [PROCESSOR_EVENT]: ProcessorEvent.metric } },
+            { term: { [METRICSET_NAME]: 'transaction' } },
+            { terms: { _tier: ['content', 'hot', 'warm'] } },
```

Comment:
When the jobs are started from within the APM UI, what start time is used for the job? Unless you are running them over the full time range of data available, this `_tier` filter isn't needed. Assuming the start date is close to `now`, the time range part of the datafeed query will ensure the cold / frozen tiers won't be searched.